src/ssl.c

```diff
@@ -48163,7 +48163,7 @@ int wolfSSL_PEM_write_bio_X509_REQ(WOLFSSL_BIO *bp, WOLFSSL_X509 *x)
     int derSz;
     int ret;
 
-    WOLFSSL_ENTER("wolfSSL_PEM_write_bio_X509_REQ()");
+    WOLFSSL_ENTER("wolfSSL_PEM_write_bio_X509()");
```

Comment:
This seems wrong. Function is `wolfSSL_PEM_write_bio_X509_REQ`.

Reply:
Thanks!  This change needed to happen a few functions below this one, adjusted.